source/development/packaging/generate-wazuh-splunk-app.rst

```diff
@@ -32,7 +32,7 @@ Execute the ``generate_wazuh_app.sh`` script, with the different options you des
 
   Usage: ./generate_wazuh_splunk_app.sh [OPTIONS]
 
-      -b, --branch <branch>     [Required] Select Git branch or tag e.g. v3.11.4-7.3.0
+      -b, --branch <branch>     [Required] Select Git branch or tag e.g. v|WAZUH_LATEST|-7.3.0
```

Comment:
7.3.0 must be changed by splunk latest which is 8.0.1

Reply:
Fixed.